src/main/resources/standardTestCases/bad_docker_name.test

```diff
@@ -0,0 +1,7 @@
+name: bad_docker_name
+testFormat: workflowfailure
+
+files {
+  wdl: bad_docker_name/bad_docker_name.wdl
+  inputs: hello/hello.inputs
+}
```

Comment:
@ruchim It turns out that JES and Local don't even use the same error messages 😡 

I have a feeling we're going to start seeing a lot of these as we move these tests across :rage4: